src/ds/defines.h

```diff
@@ -168,4 +166,6 @@ namespace snmalloc
 #else
   static constexpr bool CHECK_CLIENT = false;
 #endif
+  template <typename... Args>
+  void UNUSED(Args&& ... unused [[maybe_unused]]) {}
```

Comment:
Out of interest, does the following also work?
```C++
template <typename... Args>
inline void UNUSED(Args&& ...) {}
```
It is definitely unused, so I think you can just omit giving the argument pack a name.

But I love the change, awesome trick.